tests/osaka/eip7883_modexp_gas_increase/test_modexp_thresholds.py

```diff
@@ -440,26 +616,34 @@ def create_modexp_variable_gas_test_cases():
     # │ IC9 │  S   │  =  │  B   │  N/A  │   N/A   │ Zero modulus handling                         │
     # │ IC10│  S   │  =  │  B   │ False │  4080   │ Power-of-2 boundary with high bit             │
     # └─────┴──────┴─────┴──────┴───────┴─────────┴───────────────────────────────────────────────┘
-    for base, exponent, modulus, expected_result, test_id in test_cases:
+    for base, exponent, modulus, expected_result, gas_usage, test_id in test_cases:
         yield pytest.param(
             ModExpInput(base=base, exponent=exponent, modulus=modulus),
             bytes.fromhex(expected_result),
+            gas_usage,
             id=test_id,
         )
 
 
 @pytest.mark.parametrize(
-    "modexp_input,modexp_expected",
+    "modexp_input,modexp_expected,gas_usage",
     create_modexp_variable_gas_test_cases(),
 )
+@EIPChecklist.Precompile.Test.InputLengths.Zero()
+@EIPChecklist.GasCostChanges.Test.GasUpdatesMeasurement()
 @pytest.mark.valid_from("Berlin")
 def test_modexp_variable_gas_cost(
     state_test: StateTestFiller,
+    precompile_gas: int,
+    gas_usage: int,
     pre: Alloc,
     tx: Transaction,
+    fork: Fork,
     post: Dict,
 ):
     """Test ModExp variable gas cost."""
+    if fork >= Osaka:
+        assert (gas_usage is None) or (precompile_gas >= gas_usage), "inconsistent gas usage"
```

Comment:
```suggestion
    if fork >= Osaka:
        assert (gas_usage is None) or (precompile_gas >= gas_usage), "inconsistent gas usage"
```
Do we need this? I removed it and all the tests pass?

Reply:
We need this! Since the test are starting from previous fork (`Berlin` or `London`), but the `gas_usage` is calculated based on `Osaka` only.

I add the last commit since I encounter CI failure without the if-else statement -> you could check this [efe9de9](https://github.com/ethereum/execution-spec-tests/pull/2098/commits/efe9de92eeb68d8479eac92ac09007e986b22745), it is the version without the change

Reply:
Ahh, I understand now! Maybe we should add a comment here:
```
if fork >= Osaka: # Check that gas used defined in table is accurate
```